src/components/PersonCard/PersonCard.tsx

```diff
@@ -0,0 +1,138 @@
+import {
+  Box,
+  Card,
+  CardActions,
+  CardContent,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import clsx from 'clsx';
+import { FC } from 'react';
+import * as React from 'react';
+import { Picture } from '../Picture';
+import { Link } from '../Routing';
+
+const useStyles = makeStyles(({ palette, spacing }) => ({
+  personCard: {
+    maxWidth: '247px',
+    paddingTop: spacing(1),
+  },
+  cardContent: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  personName: {
+    paddingBottom: spacing(0.5),
+  },
+  personImage: {
+    width: '86px',
+    height: '86px',
+    paddingBottom: spacing(2),
+  },
+  roleName: {
+    marginBottom: spacing(1),
+  },
+  organizationName: {
+    marginBottom: spacing(0.25),
+  },
+  statusCircle: {
+    height: '8px',
+    width: '8px',
+    marginRight: spacing(1),
+    borderRadius: '50%',
+    display: 'inline-block',
+  },
+  cardActions: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    padding: spacing(2),
+    border: '0.5px solid #e0e0e0',
+  },
+  activeColor: {
+    backgroundColor: palette.primary.main,
+  },
+  notActiveColor: {
+    // TODO not sure exactly what color this should be
+    backgroundColor: palette.error.main,
+  },
+}));
+
+export interface PersonCardProps {
+  name: string;
+  organization: string;
+  role: string;
+  active: boolean;
+  personRoute: string;
+  imageSource: string;
+}
+
+export const PersonCard: FC<PersonCardProps> = ({
+  name,
+  organization,
+  role,
+  active,
+  personRoute,
+  imageSource,
+}) => {
+  const {
+    personCard,
+    personName,
+    statusCircle,
+    cardContent,
+    personImage,
+    roleName,
+    organizationName,
+    cardActions,
+    activeColor,
+    notActiveColor,
+  } = useStyles();
```

Comment:
I've held off on commenting on this until now, because it feels petty. But typically we don't de-structure the css classes. It helps to not populate the variable namespace and make the quick distinction between them and props.
It does feel arbitrary though since we do destructure props. The only thing I can say is it would probably help to be consistent across the codebase.